src/CMakeLists.txt

```diff
@@ -137,7 +137,11 @@ add_library(parthenon
   parameter_input.cpp
   parthenon_manager.cpp
 )
+
 target_compile_features(parthenon PUBLIC cxx_std_14)
+if (CMAKE_CXX_COMPILER_ID STREQUAL "XL")
```

Comment:
I am not a `cmake` expert, and I know everything is a string for `cmake`. Still, this makes me uncomfortable. Is this always going to work?

Reply:
I don't understand - what makes you uncomfortable?

Reply:
Just worried that "XL" might not always be the right string to match. Which is being discussed in the broader thread. (My bad, I didn't notice the conversation with @nmsriram )